app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1> <%= @recipe.name %> </h1>
+<p>Complexity: <%= @recipe.complexity %> </p>
+<p>Genre: <%= @recipe.genre %> </p>
+<% @recipe_ingredients.each do |ingredient| %>
+  <p> <%= ingredient.name %>: <%= ingredient.cost %> </p>
+<% end %> 
+<p> Total Cost: <%= @recipe_ingredients_cost %> </p>
```

Comment:
Talked about this a couple of times already, but there is a much better way to grab the recipe ingredients and total cost of ingredients.